actionpack/lib/action_controller/log_subscriber.rb

```diff
@@ -60,9 +60,9 @@ def unpermitted_parameters(event)
       class_eval <<-METHOD, __FILE__, __LINE__ + 1
         def #{method}(event)
           return unless logger.info? && ActionController::Base.enable_fragment_cache_logging
-          key_or_path = event.payload[:key] || event.payload[:path]
+          key         = ActiveSupport::Cache.expand_cache_key(event.payload[:key] || event.payload[:path])
```

Comment:
Agree. I'd rather make it a direct setting for Rails.cache, but that's for a later PR.